src/bms/parse/model/obj.rs

```diff
@@ -198,31 +205,31 @@ impl Ord for ScrollingFactorObj {
     }
 }
 
-/// An object to change spacing factor between notes with linear interpolation.
+/// An object to change speed factor of notes (a.k.a. spacing factor between notes) with linear interpolation.
```

Comment:
```suggestion
/// An object to change spacing factor among notes with linear interpolation.
```
According to [https://bemuse.ninja/project/docs/bms-extensions/](Bemuse's extension doc), this object changes the notes spacing, not scrolling speed. So I'd like to avoid "speed factor" to describe the meaning of this object.